lib/matplotlib/dates.py

```diff
@@ -629,13 +629,13 @@ class AutoDateFormatter(ticker.Formatter):
     format string.  The default looks like this::
 
         self.scaled = {
-           365.0  : '%Y',
-           30.    : '%b %Y',
-           1.0    : '%b %d %Y',
-           1./24. : '%H:%M:%S',
-           1. / (24. * 60.): '%H:%M:%S.%f',
-           }
-
+           DAYS_PER_YEAR          : '%Y',
+           DAYS_PER_MONTH         : '%b %Y',
+           1.0                    : '%b %d %Y',
+           1. / HOURS_PER_DAY     : '%H:%M',
+           1. / SEC_PER_DAY       : '%H:%M:%S',
+           1. / MUSECONDS_PER_DAY : '%H:%M:%S.%f'
```

Comment:
I would say it makes the documentation more clear like this? Also, expanding MUSECONDS_PER_DAY would result in `1. / (24 * 60 * 60 * 1000000)`, which struck me as overly complicated. But I'm happy to change back to numbers if you think that would be better?